### PR TITLE
Change name of step variable to allow reset to work

### PR DESCRIPTION
**Pull request by pgrinaway**
- [x] Implement feature / fix bug
 - [x] Add [tests](https://github.com/choderalab/openmmtools/tree/master/openmmtools/tests)

### CODE REVIEW

**Comment by pgrinaway** [comment]
Looks like some [weirdness](https://travis-ci.org/choderalab/openmmtools/jobs/447164641#L3431) in the build here....

**Comment by jchodera** [comment]
@andrrizzi likely has insight into that segfault. It was previously claimed to be stochastic, but it seems to happen quite often.

**Comment by jchodera** [comment]
I've restarted the build just in case.

File: openmmtools/integrators.py [hunk 1]
```diff
@@ -1856,19 +1856,19 @@ def __init__(self, *args, **kwargs):
...
-        self.addGlobalVariable("first_step", 0)
+        self.addGlobalVariable("step", 0)
 
     def _add_integrator_steps(self):
         self.addComputeGlobal("perturbed_pe", "energy")
         # Assumes no perturbation is done before doing the initial MD step.
-        self.beginIfBlock("first_step < 1")
-        self.addComputeGlobal("first_step", "1")
+        self.beginIfBlock("step < 1")
+        self.addComputeGlobal("step", "1")
```
Comment (jchodera) [inline openmmtools/integrators.py]: To match the other integrators, why not increment `step` each step?

Comment (pgrinaway) [reply openmmtools/integrators.py]: ok

**Pull request merged**